source/monitoring-and-logging.txt

```diff
@@ -0,0 +1,237 @@
+.. _kafka-monitoring-logging:
+
+======================
+Monitoring and Logging
+======================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+
+
+
+Sink Connector JMX Metrics
+--------------------------
+
+Use the metrics in the following table to monitor the behavior of your sink
+connector through {+jmx-long+} (JMX):
+
+.. tip:: JMX Attributes
+
+   {+jmx-hover+} represents an individual metric as an attribute of an ``MBean``.
+   To learn more about attributes and ``MBeans``, see the
+   `Standard MBeans Tutorial <https://docs.oracle.com/javase/tutorial/jmx/mbeans/standard.html>`__
+   from Oracle.
+
+.. list-table::
+   :header-rows: 1
+   :widths: 40 60
+
+   * - Attribute Name
+     - Description
+
+   * - | **records**
+       | 
+     - | The total number of Kafka records a MongoDB sink task received.
+
+   * - | **records-successful**
+       | 
+     - | The total number of Kafka records a MongoDB sink task successfully
```

Comment:
Question: since the line block ("|") isn't used to preserve a linebreak in the Description column, all the cells in this column are equivalent without them. Is there some other reason they need to be included?

Reply:
removed line breaks